### PR TITLE
Fixed condor_q retries, added cancellation

### PR DESCRIPTION
**Pull request by bio-boris**

### CODE REVIEW

File: deployment/misc/condor_rm.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+batchName=$1;
+condorCommand="condor_rm  -constraint 'JobBatchName==\"$batchName\"'";
+eval $condorCommand
```
Comment (bio-boris) [inline deployment/misc/condor_rm.sh]: New File

File: src/us/kbase/common/utils/CondorUtils.java [hunk 2]
```diff
@@ -30,7 +27,7 @@ private static File createCondorSubmitFile(String ujsJobId, String token, String
         String arguments = String.join(" ", args);
         List<String> csf = new ArrayList<String>();
         csf.add("universe = vanilla");
-        csf.add("accounting_group = sychan");
+        csf.add(String.format("accounting_group = %s", token.getUserName()));
```
Comment (bio-boris) [inline src/us/kbase/common/utils/CondorUtils.java]: User Groups

File: src/us/kbase/common/utils/CondorUtils.java [hunk 3]
```diff
@@ -134,29 +129,36 @@ public static String submitToCondorCLI(String ujsJobId, String token, String cli
...
-        if (result.contains(attribute)) {
-            try {
-                ObjectMapper mapper = new ObjectMapper();
-                List<Map<String, Object>> myObjects =
-                        mapper.readValue(result, new TypeReference<List<Map<String, Object>>>() {
-                        });
-                return myObjects.get(0).get(attribute).toString();
-            } catch (com.fasterxml.jackson.databind.JsonMappingException e) {
-                return null;
+        while (result == null && retries > 0) {
```
Comment (bio-boris) [inline src/us/kbase/common/utils/CondorUtils.java]: New Retry Logic

File: dev_tools/install.sh [hunk 4]
```diff
@@ -6,7 +6,7 @@ apt-get update
 apt-get -y install git;
```
Comment (kkellerlbl) [inline dev_tools/install.sh]: Is this file for installing software locally?

**Comment by bio-boris** [comment]
Yes for Dev

File: dev_tools/install.sh (hunk 4 above)
Comment (bio-boris) [reply dev_tools/install.sh]: Yes for Dev

**Pull request merged**